src/ontology/uberon-edit.obo

```diff
@@ -43863,6 +43862,8 @@ xref: SCTID:278981008
 xref: UMLS:C0225666 {source="ncithesaurus:Terminal_Bronchiole"}
 xref: Wikipedia:Terminal_bronchiole
 is_a: UBERON:0002186 ! bronchiole
+relationship: distally_connected_to UBERON:0002188 ! respiratory bronchiole
```

Comment:
Consider if 'continuous with' is a more accurate object property.

Reply:
Thank you for this comment, Bradley. There is this related ticket in Uberon #2926 and it was treated in the last Uberon call (19/06/23) about the usage of connected to and how it is not aligned with the definitions. However, it was agreed to use this object property for the pulmonary system.